src/components/Home.style.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex !important;
+  width: 100vw;
+  flex-direction: column;
+  align-items: center
+`;
+
+export const Title = styled.p`
+  margin-top: 27px;
+  margin-bottom: 10px;
+  font-size: 38px;
+  font-family: ${(props) => props.theme.font.family.serif};
+  text-align: center
+`;
+
+export const Subtitle = styled.p`
+  text-align: center;
+  margin-top: 10px;
+  margin-bottom: 22px;
+  font-size: 16px;
+  font-family: ${(props) => props.theme.font.family.sansSerif};
+  color: ${(props) => props.theme.colors.text.lighterGray};
+`;
+
+export const Button = styled.button`
+  outline: none;
+  border: none;
+  color: white;
+  cursor: pointer;
+  background-color: ${(props) => props.theme.colors.orange};
+
+  width: 217px;
```

Comment:
A fixed width shouldn't be required here. Padding is typically enough. This way the button will be easier to reuse since the text length can vary

Reply:
Makes sense--fixed!